charts/index.yaml

```diff
@@ -3,10 +3,10 @@ entries:
   secrets-store-csi-driver:
   - apiVersion: v1
     appVersion: 0.0.12
-    created: "2020-07-17T16:17:16.643387-07:00"
+    created: "2020-07-21T17:21:06.530228-07:00"
     description: A Helm chart to install the SecretsStore CSI Driver inside a Kubernetes
       cluster.
-    digest: 7175b2b870d724e87b56d7dcacdd5da11442b73aca3648382f489bccff423522
```

Comment:
Repackaging the helm charts here with same chart version as the `--metrics-addr` flag added to the charts aren't breaking change.